writer/store_events.py

```diff
@@ -130,9 +130,9 @@ def store_event_list(data_dir, station_id, cluster, event_list):
             else:
                 logger.error("Strange event (no timestamp!), discarding event "
                              "(station: %s)" % station_id)
-        except:
```

Comment:
This does not really log exceptions very well. I got some very incomprehensible `TypeError`s when porting to python 3. Without a traceback, those are hard to track down.

I tried this: tomkooij/datastore@7cea742  but I'm hoping there is a better way.